ClientApp/src/components/superadmin/Municipality/ManageMunicipality/MunicipalityTableElements.tsx

```diff
@@ -51,7 +52,7 @@ const MunicipalityTableElements: React.FC<props> = ({
           <TextField
             disabled={openEditForm || role != "SuperAdmin"}
             placeholder={updatedMunicipality.email == "" ? "Fyll inn epost" : ""}
-            value={updatedMunicipality.email}
+            value={!!updatedMunicipality.email ? updatedMunicipality.email : ""}
```

Comment:
will this not be the same?
```suggestion
            value={updatedMunicipality.email ? updatedMunicipality.email : ""}
```

or maybe this will work?
```suggestion
            value={updatedMunicipality.email ?? ""}
```